js/src/components/google-combo-account-card/claim-ads-account/claim-ads-account.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * External dependencies
+ */
+import { useCallback, useState, useRef } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import ClaimAccountButton from '.~/components/google-ads-account-card/claim-account-button';
+import { useAppDispatch } from '.~/data';
+import useWindowFocusCallbackIntervalEffect from '.~/hooks/useWindowFocusCallbackIntervalEffect';
+import './claim-ads-account.scss';
+
+const CLAIM_RETRIES = 3;
+
+/**
+ * ClaimAdsAccount component.
+ *
+ * @return {JSX.Element} ClaimAdsAccount component.
+ */
+const ClaimAdsAccount = () => {
+	const [ claiming, setClaiming ] = useState( false );
+	const retries = useRef( CLAIM_RETRIES );
+	const { fetchGoogleAdsAccountStatus } = useAppDispatch();
+
+	const checkUpdatedAdsAccountStatus = useCallback( async () => {
+		if ( ! claiming ) {
+			return;
+		}
+
+		await fetchGoogleAdsAccountStatus()
+			.then( () => {
+				retries.current -= 1;
+				// Reset the claiming state after 3 retries.
+				if ( retries.current < 1 ) {
+					setClaiming( false );
+				}
+			} )
```

Comment:
For simplicity, I've removed all of the loading logic in [6c93d64](https://github.com/woocommerce/google-listings-and-ads/pull/2644/commits/6c93d643464e4697b6ff67b867b05e3d63b215c6) and the button will always be clickable and as long as the `ClaimAdsAccount` component is visible, it will always poll the API for updates on the account-status.